src/functions/flows/leaderboard/triggers.ts

```diff
@@ -0,0 +1,14 @@
+import { onSchedule }      from 'firebase-functions/v2/scheduler'
+import { enhanceFunction } from '../../manager'
+
+export const leaderboard = onSchedule('every day',
```

Comment:
Why is this needed? Is possible to handle this ordering using a query? ordering?